base/special/exp.jl

```diff
@@ -32,22 +32,22 @@
 # log(2)
 const LN2 = 6.931471805599453094172321214581765680755001343602552541206800094933936219696955e-01
 # log2(e)
-const LOG2E = 1.442695040888963407359924681001892137426646
+const LOG2_E = 1.442695040888963407359924681001892137426646
 
-# log(2) into upper and lower
+# log(2) into upper and lower bits
 LN2U(::Type{Float64}) = 6.93147180369123816490e-1
 LN2U(::Type{Float32}) = 6.9313812256f-1
 
 LN2L(::Type{Float64}) = 1.90821492927058770002e-10
 LN2L(::Type{Float32}) = 9.0580006145f-6
 
-# max and min arguments for exponential fucntions
-MAXEXP(::Type{Float64}) = 7.09782712893383996732e2 # log 2^1023*(2-2^-52)
-MAXEXP(::Type{Float32}) = 88.72283905206835f0      # log 2^127 *(2-2^-23)
+# max and min arguments
+EXP_MAX(::Type{Float64}) = 7.09782712893383996732e2 # log 2^1023*(2-2^-52)
```

Comment:
bikeshedding but do people prefer
`EXP_MAX`
`MAX_EXP`
or the no underscore variants of the above